lib/uniris/beacon_chain/subset.ex

```diff
@@ -85,11 +85,11 @@ defmodule Uniris.BeaconChain.Subset do
     listNodes =
       Enum.filter(P2P.list_nodes(), fn x -> :binary.part(x.first_public_key, 0, 1) == subset end)
 
-    _p2p_view_available = Enum.map(listNodes, fn x -> GenICMP.ping(x.ip) end)
+    p2p_view_available = Enum.map(listNodes, fn x -> GenICMP.ping(x.ip) end)
 
     new_registry = SlotRegistry.seal_current_slot(slot_registry, slot_time)
 
-    {:noreply, %{state | slot_registry: new_registry}}
+    {:noreply, %{state | slot_registry: new_registry, p2p_view_available: p2p_view_available}}
```

Comment:
You have to use `Map.put/3` to create the new state, as the `p2p_view_available` field is not existing yet inside the state map. This notation will not work.
This will work:


```elixir
new_state = state
    |> Map.put(:slot_registry, new_registry)
    |> Map.put(:p2p_view_available, p2p_view_available)
```